src/components/NewHathorAlert.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useRef, forwardRef, useImperativeHandle } from 'react';
+import { ReactComponent as SuccessIcon } from '../assets/images/success-icon.svg';
+
+/**
+ * NewHathorAlert - A functional alert component with an icon and customizable display duration.
+ * This component can be shown for a specified duration by calling the `show` method from its parent component.
+ *
+ * @param {Object} props - Component properties.
+ * @param {string} props.type - Defines the alert type for styling (e.g., "success", "error").
```

Comment:
We haven’t defined it yet because, until this PR, we’ve only encountered the success case. Once we come across other scenarios in Figma, we can define them as needed. This way, we align the development flow closely with Figma and avoid overthinking scenarios we haven’t encountered yet. Could it be this way?